labgrid/remote/exporter.py

```diff
@@ -220,15 +220,30 @@ def _start(self, start_params):
         assert self.child is None
         assert start_params['path'].startswith('/dev/')
         self.port = get_free_port()
-        self.child = subprocess.Popen([
-            self.ser2net_bin,
-            '-d',
-            '-n',
-            '-C',
-            '{}:telnet:0:{}:{} 8DATABITS NONE 1STOPBIT LOCAL'.format(
-                self.port, start_params['path'], self.local.speed
-            ),
-        ])
+
+        # Ser2net has switched to using YAML format at version 4.0.0.
+        _, _, version = str(subprocess.check_output(['ser2net','-v'])).split(' ')
+        if version > "4.0.0":
+            self.child = subprocess.Popen([
+                '/usr/sbin/ser2net',
+                '-d',
+                '-n',
+                '-Y',
+                'connection: &con01#  accepter: telnet, {}'.format(self.port),
+                '-Y',
+                '  connector: serialdev,{},115200n81,local'.format(start_params['path']
+                ),
+            ])
+        else:
+            self.child = subprocess.Popen([
+                '/usr/sbin/ser2net',
```

Comment:
same as above, please use `self.ser2net_bin`.

Reply:
Will do. Thanks for the comment!